jboss/container/wildfly/launch/keycloak/added/keycloak.sh

```diff
@@ -77,11 +71,11 @@ function configure_cli_keycloak() {
     local xpath="\"//*[local-name()='subsystem' and starts-with(namespace-uri(), 'urn:jboss:domain:security:')]\""
     testXpathExpression "${xpath}" "has_security_subsystem"
 
-    useLegacySecurity=false;
-
-    # In some context we want to use legacy subsystem only.
-    if [ "x${SSO_FORCE_LEGACY_SECURITY}" == "xtrue"  ] && [ "${has_security_subsystem}" -eq 0 ]; then
-      useLegacySecurity=true;
+    # No more supported
+    if [ -n "${SSO_FORCE_LEGACY_SECURITY}" ]; then
```

Comment:
I would prefer to fail in case one use this env variable. It is false by default (the behavior we want) so having the env in the config means to force the legacy security subsystem (that is no more present in the server), so not correct and the env should be fixed.